libs/ledger-live-common/src/families/cardano/js-synchronisation.ts

```diff
@@ -349,8 +352,8 @@ const makeGetAccountShape =
     return {
       id: accountId,
       xpub,
-      balance: accountBalance,
-      spendableBalance: accountBalance.minus(minAdaBalanceForTokens),
+      balance: totalBalance,
```

Comment:
I would get rid of the extra `totalBalance` variable above.

```suggestion
      balance: delegationInfo?.rewards ? utxosSum.plus(delegationInfo.rewards) : utxosSum,
```

Reply:
Done